SetupConfigure.cmake

```diff
@@ -196,7 +196,11 @@ if (NOT MUE_BUILD_AUDIO_MODULE)
 endif()
 
 if (MUE_ENABLE_AUDIO_JACK)
-    add_definitions(-DJACK_AUDIO)
+    if (OS_IS_LIN OR MINGW)
+        add_definitions(-DJACK_AUDIO)
+    else()
+        set(MUE_ENABLE_AUDIO_JACK OFF)
```

Comment:
So not for the 'regular' Windows MSVC build?

Reply:
And with using MinGW it doesn't build at all, as far as I can tell, see https://github.com/musescore/MuseScore/pull/19349#issuecomment-1717220670

Reply:
Although I must admit to not have tried again after the switch to Qt6

Reply:
Yes, no jack for native window since you need the weakapi to load jack dll.
Yeah, ages since I built on mingw too.. it could be a build target as done in musescore-3: https://github.com/lyrra/Musescore3/blob/main/.github/workflows/ci_windows.yml
But it isn't an prioritized platform anyway.